docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -30,7 +30,7 @@ When you run Cypress via `cypress run` and pass the `--record` flag, the followi
 - Videos
 - OS environment variables related to CI and git information
 
-When [Test Replay](#Test-Replay) capturing is enabled, the additional data outlined below will be sent to the Cloud. You can disable capturing: TODO link this data.
+When [Test Replay](#Test-Replay) capturing is enabled, the additional data outlined below will be sent to the Cypress Cloud. You can disable capturing: TODO link this data.
```

Comment:
probably just, Cypress Cloud. can loose "the".

Reply:
Addressed in [`5281bbf` (#5425)](https://github.com/cypress-io/cypress-documentation/pull/5425/commits/5281bbf6bae773d499bdd69b81999f88db93b871)